pkg/authorization/api/v1/types.go

```diff
@@ -112,6 +112,11 @@ type ResourceAccessReviewResponse struct {
 	UsersSlice []string `json:"users" description:"list of users who can perform the action"`
 	// GroupsSlice is the list of groups who can perform the action
 	GroupsSlice []string `json:"groups" description:"list of groups who can perform the action"`
+
+	// EvaluationError is an indication that some error occurred during resolution, but partial results can still be returned.
+	// It is entirely possible to get an error and be able to continue determine authorization status in spite of it.  This is
+	// most common when a bound role is missing, but enough roles are still present and bound to reason about the request.
+	EvaluationError string `json:"evalutionError" description:"error occurred during resolution, but partial results can still be returned"`
```

Comment:
You need to define partial.  Is partial "if I called this and I expected to get the full list, but I didn't, I can safely continue?"  Or is partial "here's as much of the results as we can show"?  If it's the latter, this needs to be a struct or something that conveys more data about it being a partial response (for one thing, people shouldn't be doing `if len(e.EvaluationError) == 0 { // happy path }`)